public/app/features/dashboard/dashgrid/LazyLoader.tsx

```diff
@@ -12,7 +10,7 @@ export interface Props {
 }
 
 export function LazyLoader({ children, width, height, onLoad, onChange }: Props) {
-  const id = useUniqueId();
+  const id = useId();
```

Comment:
I would recommend we put these changes into a seperate PR so they can be reviewed, and if needed, reverted, independently.

Reply:
It was separated See https://github.com/grafana/grafana/pull/69542
But somehow github change list is still showing it :( 
That's why I am closing this one and creating a new one.